AttendanceTracker/Domain/Activity.Domain.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Why is netcoreapp 3.1 needed? The general rule is that you target standard for class libraries since it is accessible with way more runtimes.
Else, please put Project type ClassLibrary